doc/changelog.rst

```diff
@@ -11,56 +11,125 @@ Jump to :ref:`SmartRedis Changelog <sr_changelog>`
+- Relax the required version of `typing_extensions`. (SmartSim-PR459_)
+- Addition of Python 3.11 to SmartSim. (SmartSim-PR461_)
+- Quality of life `smart validate` improvements such as seting `CUDA_VISIBLE_DEVICES`
+  environment variable within `smart validate` prior to importing any ML deps to
+  prevent false negatives on multi-GPU systems. Additionally, moving SmartRedis logs
+  from standard out to dedicated log file in the validation temporary directory as well as
+  suppress `sklearn` deprecation warning by pinning `KMeans` constructor
+  argument. Lastly, move TF test to last as TF may reserve the GPUs it uses.
+  (SmartSim-PR458_)
 - Some actions in the current GitHub CI/CD workflows were outdated. They were
   replaced with their latest versions. (SmartSim-PR446_)
 - As the Cobalt workload manager is not used on any system we are aware of,
   its support in SmartSim was terminated and classes such as `CobaltLauncher` have
   been removed. (SmartSim-PR448_)
-- The sphinx-tabs documentation extension uses a white background for the tabs component.
-  A custom CSS for those components to inherit the overall theme color has
-  been added. (SmartSim-PR453_)
+- The implementation makes use of python `contextvars.ContextVar` to store
+  experiment-specific state. The state is used to dynamically modify
+  experiment-level logging. (SmartSim-PR452_)
 - Updated SmartSim's machine learning backends to PyTorch 2.0.1, Tensorflow
   2.13.1, ONNX 1.14.1, and ONNX Runtime 1.16.1. As a result of this change,
   there is now an available ONNX wheel for use with Python 3.10, and wheels for
   all of SmartSim's machine learning backends with Python 3.11.
   (SmartSim-PR451_) (SmartSim-PR461_)
-- SmartSim can now be built and used on platforms using Apple Silicon
-  (ARM64). Currently, only the PyTorch backend is supported. Note that libtorch
-  will be downloaded from a CrayLabs github repo. (SmartSim-PR465_)
-- Separate install instructions are now provided for Mac OSX on x64 vs ARM64 (SmartSim-PR478_)
-
+- The sphinx-tabs documentation extension uses a white background for the tabs component.
+  A custom CSS for those components to inherit the overall theme color has
+  been added. (SmartSim-PR453_)
+- This PR adds concurrency groups to GitHub's CI/CD workflows, preventing
+  multiple workflows from the same PR to be launched concurrently.
+  (SmartSim-PR439_)
+- Torch changed something in their indexing when trying to install from
+  their provided wheels. This updates the `pip install` command within
+  `smart build` to ensure that the appropriate packages can be found.
+  (SmartSim-PR449_)
+
+.. _SmartSim-PR482: https://github.com/CrayLabs/SmartSim/pull/482
+.. _SmartSim-PR479: https://github.com/CrayLabs/SmartSim/pull/479
+.. _SmartSim-PR475: https://github.com/CrayLabs/SmartSim/pull/475
+.. _SmartSim-PR485: https://github.com/CrayLabs/SmartSim/pull/485
+.. _SmartSim-PR473: https://github.com/CrayLabs/SmartSim/pull/473
```

Comment:
Okay, very incredibly pedantic request, but I am insane and need things to be in order, so you're  gonna have to bear with me: can we sort this list of links, hahaha?

Reply:
yes! I put it in order of merged but can switch